### PR TITLE
VZ-6113 Changes for parsing jaeger logs by fluentd

### PR DESCRIPTION
**Pull request by anajosep**
VZ-6113 - Changes in fluentd config to parse logs from Jaeger Operator and Jaeger Components (query, ui, collector)

### CODE REVIEW

**Review by fredtibbitts — APPROVED** [review]
I can't comment on the Jaeger parsing specifics but the changes generally seem fine to me.

File: tests/e2e/logging/system/system_logging_test.go [hunk 1]
```diff
@@ -512,6 +518,26 @@ func validateNodeExporterLogs() bool {
 		noExceptions)
 }
 
+func validateJaegerCollectorLogs() bool {
+	return validateElasticsearchRecords(
+		logLevelElasticsearchRecordValidator,
+		func() (string, error) { return pkg.GetOpenSearchSystemIndex(monitoringNamespace) },
```
Comment (desagar) [inline tests/e2e/logging/system/system_logging_test.go:524]: This namespace (which is declared with value "monitoring" at the top of this file), no longer exists.

Comment (davisgomes) [reply tests/e2e/logging/system/system_logging_test.go:524]: Same comment from me.

File: platform-operator/helm_config/charts/verrazzano-fluentd/templates/fluentd-config-configmap.yaml [hunk 2]
```diff
@@ -707,6 +707,35 @@ data:
...
+    # filter to parse jaeger operator and jaeger resource log files
+    <filter kubernetes.**jaeger-operator-**verrazzano-monitoring_jaeger-**>
+      @type parser
+      @id jaeger
+      key_name log
+      reserve_data true
+      emit_invalid_record_to_error true
+      <parse>
+        @type multi_format
+        # zap log format for jaeger components
```
Comment (kminder) [inline platform-operator/helm_config/charts/verrazzano-fluentd/templates/fluentd-config-configmap.yaml:719]: The reference to jaeger here should be capitalized.

File: tests/e2e/logging/system/system_logging_test.go (hunk 1 above)
Comment (anajosep) [reply tests/e2e/logging/system/system_logging_test.go:524]: This namespace is using for the getting the index name in OS. From v1.3, data streams are being used and all the system logs are stored in a single data stream `verrazzano-system` and for older versions, the old index name would be used which is `verrazzano-system-monitoring`. `monitoring` is the right value to be used here. For v1.3+ this value is ignored and it always returns the index to look for as `verrazzano-system`

`if usingDataStreams {
		return "verrazzano-system", nil
	}
`

File: platform-operator/helm_config/charts/verrazzano-fluentd/templates/fluentd-config-configmap.yaml (hunk 2 above)
Comment (anajosep) [reply platform-operator/helm_config/charts/verrazzano-fluentd/templates/fluentd-config-configmap.yaml]: Corrected the commented lines to have `Jaeger`

**Review by fredtibbitts — APPROVED** [review]

**Pull request merged**